include/libcyphal/transport/can/delegate.hpp

```diff
@@ -152,13 +170,20 @@ class TransportDelegate
         ServicePortRemoved
     };
 
+    // NOSONAR below b/c Sonar's cpp:S134 conflicts with AUTOSAR A12-1-2
```

Comment:
Conflicting rules should be turned off in Sonar

Reply:
They are not always conflicting.

Reply:
OK, I agree with "erode" theme - I'll try disable this tidy rule.